services/github_publisher.py

```
@@ -84,86 +86,118 @@ def publish_markdown(
+            else:
+                # Create new feature branch name
+                feature_branch = f"blog/{path.replace('/', '_').replace('.md', '')}"
+                target_branch = feature_branch
+                logger.info(f"Creating new PR with feature branch: {feature_branch}")
+            
+            logger.info(f"Target branch for commit: {target_branch}")
+        
+        # Publish assets if requested
+        asset_results = []
+        if include_assets and assets_info:
+            asset_results = self._publish_assets(
```

Ruff (0.12.2)

Local variable `asset_results` is assigned to but never used

Remove assignment to unused variable `asset_results`

(F841)